src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/entity_search_input.html.twig

```diff
@@ -187,12 +171,17 @@ these widgets are the prototype templates (that are handled by js later) and the
 on page load because of disablingSwitch component behavior. But they must not be disabled
 or else will not be submitted to the form, so we remove "disabled" attribute if it exists.
 #}
-{% block searched_customer_id_customer_widget %}
+{% block searched_entity_item_hidden_widget %}
   {% set attr = attr|filter((value, key) => key != 'disabled') %}
   {{ block('hidden_widget') }}
```

Comment:
not sure if we always must know about the widget type, or is there some syntax to render the type whichever it would be rendered by default? so we could somehow homogenize them all and render with one block? (instead of rendering the same over and over again for every type - `hidden | image-preview | text-preview`)